File: week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    //Use to take input from the user
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    //Condition check the range of number should be between 1 to 8

    //Iterate through height
    for (int i = 1; i <= height; i++)
    {
        //Print spaces in decreasing order
        for (int j = i; j < height; j++)
        {
            printf(" ");
        }

        //Print bricks in increasing order
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        //print spaces between 2 pyramids
        printf("  ");

        //Print bricks for the 2nd pyramid
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week 6/sentimental/mario-more/mario.py
from cs50 import get_int

# Immitating do while loop like we use in c
while True:
    # Taking an int input from the user
    height = get_int("Height: ")

    # Condition to check whether it is between 1 to 8 inclusive
    if 0 < height < 9:
        break

# Main code
for i in range(1, height + 1):
    # To print spaces
    j = height - i

    # Printing Spaces
    print(" " * j, end="")

    # Printing bricks
    print("#" * (i), end="  ")

    # Printing second blocks
    print("#" * (i))
